pym/bob/cmds/build.py

```diff
@@ -733,6 +733,7 @@ def _cookCheckoutStep(self, checkoutStep, depth):
             checkoutExecuted = False
             checkoutState = checkoutStep.getScmDirectories().copy()
             checkoutState[None] = checkoutDigest
+            checkoutState['checkoutAsserts'] = checkoutStep.getCheckoutAsserts().copy()
```

Comment:
That looks odd. There should be no need to treat checkout asserts as pseudo SCM.